src/main/java/com/easypost/easyvcr/requestelements/Response.java

```diff
@@ -52,256 +37,6 @@ public final class Response extends HttpElement {
      */
     private URI uri;
 
-    /**
-     * Build a CloseableHttpResponse out of this Response.
```

Comment:
Why was this whole bit ripped out?

Reply:
Was never used, it was proactive work to add support for Apache's version of HTTP client to this VCR (a plan that has since been abandoned)